src/plugins/data/server/search/search_service.ts

```diff
@@ -76,17 +80,21 @@ export class SearchService implements Plugin<ISearchSetup, ISearchStart> {
   public start(): ISearchStart {
     return {
       getSearchStrategy: this.getSearchStrategy,
-      search: this.search,
+      search: (context: RequestHandlerContext, searchRequest: IEsSearchRequest, options: any) => {
```

Comment:
I think the right type for `options` should be `Record<string, any>`

Reply:
:+1: